portal/components/react-admin/base/resources/teachers.js

```diff
@@ -282,20 +282,26 @@ export const TeacherEdit = (props) => {
           </div>
           
           <span className={classes.heading}>Update Status</span>
-          <div className={`${classes.grid}`}>
-            {/* <SelectInput
-              source="delivery_status"
-              label="Grades Taught"
-            />
-            <SelectInput
-              source="delivery_status"
-              label="Subjects Taught"
-            /> */}
+          <div className={`${classes.grid} ${classes.fullWidthGrid}`}>
             <SelectInput
               source="account_status"
               label="Account Status"
               choices={config.statusChoices}
             />
+            <FormDataConsumer>
+              {({ formData, ...rest }) =>
+                formData?.account_status === "DEACTIVATED" ? (
```

Comment:
Can we change this be picked from the [config](https://github.com/Samagra-Development/x-admin/blob/release/hp/portal/components/config.js#L139) file?